controlplane/auth.go

```diff
@@ -7,19 +7,40 @@ import (
 )
 
 type AuthInfo struct {
-	Service        string
-	WorkspaceToken string
-	InstanceID     string
+	Service         string
+	ConnectionToken string
+	InstanceID      string
 }
 
 type authService struct {
 	authInfo AuthInfo
 	proto.UnimplementedDPAuthServiceServer
 }
 
+func (a *authService) GetConnectionToken(ctx context.Context, request *proto.GetConnectionTokenRequest) (*proto.GetConnectionTokenResponse, error) {
```

Comment:
To make DeepSource happy. Same in the other method.

```suggestion
func (a *authService) GetConnectionToken(_ context.Context, _ *proto.GetConnectionTokenRequest) (*proto.GetConnectionTokenResponse, error) {
```